### ISSUE TITLE
Explain how to disable DebugClassLoader when using phpunit-bridge 4.2

### ISSUE THREAD

**Issue by nicolas-grekas** [symfony/symfony-docs#10360]
And tell meanwhile that adding the Debug component will provide extra checks provided by DebugClassLoader by default.

Introduced in https://github.com/symfony/symfony/pull/28412

In phpunit.xml.dist:
```xml
    <listeners>
        <listener class="Symfony\Bridge\PhpUnit\SymfonyTestsListener">
            <arguments>
                <array>
                    <element key="debug-class-loader"><integer>0</integer></element>
...
```

**Comment by MarioBlazek** [comment]
This one can be closed, right @nicolas-grekas ?

**Comment by nicolas-grekas** [comment]
Why? I just opened it. Did anyone do the doc-PR already?

**Comment by MarioBlazek** [comment]
@nicolas-grekas, ah sorry, missed that. Can I do PR ?

**Comment by nicolas-grekas** [comment]
Sure!

**Comment by javiereguiluz** [comment]
Fixed by #10364.
